### PR TITLE
scio 0.2.0 (new formula)

### PR DESCRIPTION
**Pull request by nevillelyh**

### CODE REVIEW

**Comment by mattnworb** [comment]
👍

File: scio.rb [hunk 1]
```diff
@@ -0,0 +1,30 @@
+class Scio < Formula
+  desc "Scala API for Google Cloud Dataflow and Apache Beam"
+  homepage "https://github.com/spotify/scio/"
+  url "https://github.com/spotify/scio/releases/download/v0.2.0/scio-repl-0.2.0.jar"
+  sha256 "220e03dfd866d8475fc7943f60376ea4fe0862c3520674e46ef8262a09d1756c"
```
Comment (ravwojdyla) [inline scio.rb]: Don't you need `version` here?

File: scio.rb [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+
+  bottle :unneeded
+
+  def install
```
Comment (ravwojdyla) [inline scio.rb]: You could include this script as part of assembly jar, and only thing you would do here would be to remove `.jar` suffix. Also would make the `homebrew` people happy.

File: scio.rb (hunk 1 above)
Comment (nevillelyh) [inline scio.rb]: According to `brew audit` they figure it out from the URL with ruby+regex magic.

**Pull request merged**